packages/zowe-explorer-api/src/vscode/ui/WebView.ts

```diff
@@ -43,35 +61,65 @@ export class WebView {
      * @param context The VSCode extension context
      * @param onDidReceiveMessage Event callback: called when messages are received from the webview
      */
-    public constructor(
-        title: string,
-        webviewName: string,
-        context: ExtensionContext,
-        onDidReceiveMessage?: (message: object) => void | Promise<void>,
-        retainContext?: boolean
-    ) {
+    public constructor(title: string, webviewName: string, context: ExtensionContext, opts?: WebViewOpts) {
```

Comment:
Is this considered a breaking change? If so, should we add it to the changelog?

Reply:
Addressed in ffc40b08